packages/styles/scss/components/link/_link.scss

```diff
@@ -44,7 +43,7 @@
 
     &:focus,
     &.#{$prefix}--link--focus {
-      @include focus-outline;
+      @include focus-outline('border-small');
```

Comment:
In the future it would be great, to have a `noFocus` option to disable the focus if needed.